src/components/orgs/Coupon/List.tsx

```diff
@@ -116,6 +121,17 @@ const CouponList: React.FC<ICouponListProps> = ({ supplierId }) => {
     }
   };
 
+  const CommomUserNavigation = useUserRoles().isSupplier() ? (
+    ""
+  ) : (
+    <FloatButtonNav
+      label="buscar cupons"
+      backgroundStyle="main"
+      icon={SupplierICon}
+      href={"/"}
+    />
+  );
+

```

Comment:
- Is this a good structure? Returning an empty string? It seems like a "workaround/gambiarra" to me; Let's find another way
- +1 protein bar for the name OR the using in line:145. If you pretend to use as a COMPONENT, use <NavigationButton /> (this name also would be clearer)